recipes/ncbi-amr/compile-fix.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/Makefile b/Makefile
+index 1417b54..99a24a4 100644
+--- a/Makefile
++++ b/Makefile
+@@ -27,7 +27,8 @@ CXX=g++
+ COMPILE.cpp= $(CXX) $(CPPFLAGS)  -c
```

Comment:
Make sure the `Makefile` doesn't set CPPFLAGS to something weird. If it does, use `make CPPFLAGS="$CXXFLAGS ...(additional flags)..."`

Reply:
(Technically, CPP is in Make land C-Pre-Processor, while CXX is C++, so CPPFLAGS are things like `-I` and `-D` controlling the preprocessor, and CXXFLAGS are things like `-fopenmp`. But then, lots of Makefiles get this mixed up...)